operation.go

```diff
@@ -303,13 +303,13 @@ func (operation *Operation) parseAndExtractionParamAttribute(commentLine, object
 			if err != nil {
 				return err
 			}
-		case "maxinum":
+		case "maximum":
```

Comment:
I think u can just use 

```go
 case "maxinum":
 case "maximum":
        // rest of code here
```

instead of having the code 2 times :)

Reply:
I'm not sure what you mean. Go's case statements don't work like that, they're comma separated, but this is switching on the key of the regex, which I modified to accept either spelling, I didn't do anything two times.

Reply:
To be clear, I didn't add anything, I just modified the existing framework. The "maximum" here is a switched value on the key of the `regexAttributes` map above. The only actual required change to fix this is the modification of the regexes on lines 278 and 280, the rest are making the remaining code consistent in terms of spelling. If you would prefer to keep the typos in these case statements, that can be done, but I just figured as long as I was fixing the typo, might as well fix it everywhere.

Reply:
oh i'm sorry i missread that. guess i shouldn't review PRs before my first coffee! sorry again!